src/components/settings/SettingsNodeSelector.js

```diff
@@ -129,7 +129,7 @@ class SettingsNodeSelector extends Component {
 	};
 
 	isValidNode = (node) => {
-		return /^(https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)$/.test(node);
+		return /^(https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,8}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)$/.test(node);
```

Comment:
Will not work with custom domains

Reply:
You are right, let's remove this restriction